lib/IRGen/IRGenSIL.cpp

```diff
@@ -1222,6 +1222,10 @@ class IRGenSILFunction :
     auto e = getLoweredExplosion(i->getOperand());
     setLoweredExplosion(i, e);
   }
+  void visitDropDeinitInst(DropDeinitInst *i) {
+    auto e = getLoweredExplosion(i->getOperand());
+    setLoweredExplosion(i, e);
+  }
```

Comment:
True. This could be an unreachable.
I just copied from `visitMoveValueInst`.
But you are right, we should replace both visit functions with unreachable